### PR TITLE
Add logging documentation

### PR DESCRIPTION
**Pull request by monthonk**
*Description of changes:*

First pass at logging documentation.

Currently, there is no documentation about how to get the logs for troubleshooting at all. This should provides basic guidance for that.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/LOGGING.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Logging
+
+There are two types of logging you can use for troubleshooting. The first is CSI Driver itself and the other is Mountpoint logs.
+
+## CSI Driver logs
+
+By default, CSI Driver logs are written to pod’s `stderr` and those are captured by Kubernetes and may be retrieved with a corresponding API call:
```
Comment (muddyfish) [inline docs/LOGGING.md:7]: are written to the _driver_ pod's `stderr`

File: docs/LOGGING.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Logging
+
+There are two types of logging you can use for troubleshooting. The first is CSI Driver itself and the other is Mountpoint logs.
```
Comment (muddyfish) [inline docs/LOGGING.md:3]: The first set logs are from the CSI Driver, and the other are from Mountpoint.

File: docs/LOGGING.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Mountpoint logs
+
+Mountpoint logs are written in a node which your application pods are running. The location of the logs may vary by your operating systems, but they usually are written to host’s systemd journal. To fetch these logs, first you need to find the pod UID and node name for the pod.
+
+    POD_NAME=s3-app
+    kubectl get pods ${POD_NAME} -o jsonpath='{.metadata.uid}'
+    kubectl get pods ${POD_NAME} -o=custom-columns=NODE:.spec.nodeName
+
+Note down the pod UID and node name. Then connect to the Kubernetes node with the name you got. If you are using EKS with EC2 instances as worker nodes you can find more details about how to connect to your nodes in [AWS documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/connect-to-linux-instance.html).
```
Comment (dannycjones) [inline docs/LOGGING.md:22]:
```suggestion
Note down the pod UID and node name.

Next, connect to the Kubernetes node with the name discovered in the previous step. If you are using EKS with EC2 instances as worker nodes, you can find more details about how to connect to your nodes in the [AWS documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/connect-to-linux-instance.html).
```

File: docs/LOGGING.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+Once you are inside, use the UID from previous step to find corresponding Mountpoint logs.
+
+    POD_UID=f1015626-32ed-46e6-9634-c739c3a31312
+    PV_NAME=s3-pv
+    MOUNT_PID=$(ps -ef | grep "$POD_UID.*$PV_NAME" | grep -v "grep" | awk '{print $2}')
+    UNIT=$(systemctl status $MOUNT_PID | grep --only-matching "mount-s3-.*\.service" | tail -1)
+    journalctl --unit $UNIT
```
Comment (dannycjones) [inline docs/LOGGING.md:30]:
```suggestion
Once you have a session on the node itself, use the UID noted earlier to find the corresponding Mountpoint logs.

    POD_UID=f1015626-32ed-46e6-9634-c739c3a31312
    PV_NAME=s3-pv
    MOUNT_PID=$(ps -ef | grep "$POD_UID.*$PV_NAME" | grep -v "grep" | awk '{print $2}')
    UNIT=$(systemctl status $MOUNT_PID | grep --only-matching "mount-s3-.*\.service" | tail -1)
    journalctl --unit $UNIT
```

File: docs/LOGGING.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+See more details about Mountpoint logging in [Mountpoint documentation](https://github.com/awslabs/mountpoint-s3/blob/main/doc/LOGGING.md).
```
Comment (dannycjones) [inline docs/LOGGING.md:32]:
```suggestion
For more details about Mountpoint logging and the configuration options available, please refer to [Mountpoint's logging documentation](https://github.com/awslabs/mountpoint-s3/blob/main/doc/LOGGING.md).
```

**Review by dannycjones — COMMENTED** [review]
Nice! Thanks for kicking this off, Monthon.

I have some really nitpicky comments just because this is user documentation.

I'm also wondering how we can make these instructions simpler, but this is a great starting point.

**Comment by monthonk** [comment]
Thanks both for reviewing. I agree that we should make it simpler, this is only a starting point for the doc.

**Review by dannycjones — COMMENTED** [review]
Looks good, I'll spin up a cluster to check the steps work on a fresh cluster and then I am happy to merge.

**Review by dannycjones — CHANGES_REQUESTED** [review]
I tried out the instructions on a cluster, I have a few suggestions

File: docs/LOGGING.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+There are two types of logging you can use for troubleshooting. The first set of logs are from the CSI Driver, and the other are from Mountpoint.
+
+## CSI Driver logs
+
+By default, CSI Driver logs are written to the driver pod’s `stderr` and those are captured by Kubernetes and may be retrieved with a corresponding API call:
+
+    POD_NAME=s3-app
+    NODE_NAME=$(kubectl get pod ${POD_NAME} -o=custom-columns=NODE:.spec.nodeName | awk 'FNR == 2 {print}')
+    DRIVER_POD=$(kubectl get pods -n kube-system --field-selector spec.nodeName=${NODE_NAME} -o=custom-columns=NAME:.metadata.name | grep s3-csi-node)
+    kubectl logs ${DRIVER_POD} -n kube-system --container s3-plugin
```
[the remark this answers is not in the document]
Comment (dannycjones) [reply docs/LOGGING.md:12]: I just updated above to take the first entry (which matches the original `awk`). Not quite as simple now, but hopefully `tail` is more familiar for most than `awk`.

**Review by muddyfish — APPROVED** [review]

**Review by dannycjones — APPROVED** [review]
LGTM, thanks @monthonk

**Pull request merged**